components/member-card/card.module.scss

```diff
@@ -79,6 +79,58 @@
   font-size: 0.88rem;
 }
 
+.walletContainer {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+  border-radius: 2px;
+  width: 150px;
+  height: 70px;
+  padding: 10px 5px 2px;
+  margin-bottom: 8px;
+  margin-left: 6px;
+  background-color: #EEEEEE;
+
+  .coinData {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .coinData>p {
+    height: 2px;
+  }
+
+  .coinData>p:first-child {
+    font-size: 12px;
```

Comment:
Let's use `rem` for font-size please?
What if someone increases their system font size to say 32px? Your text would still be locked at 12px which would appear way too small

Reply:
Sure I'll add that.